.gitignore

```diff
@@ -29,3 +29,6 @@ joern-cli/workspace
 **/astgen-linux
 **/php2cpg/bin/vendor/
 **/php2cpg/bin/composer*
+
+
+.DS_Store
```

Comment:
this doesn't hurt but should rather go into your `~/.gitignore`, since it's specific to your personal setup, in this case MacOS.

Reply:
This is removed.